unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -92,6 +92,35 @@
         are not affected.
       </description>
     </request>
+
+    <event name="new_output" since="3">
+      <description summary="notify of a new wl_output">
+        This event indicates that a new output has been created. All clients
```

Comment:
It would be nice to allow multiple outputs to be advertised before sending this event. (Could be useful when plugging a DisplayPort MST hub for instance.)

Reply:
Agreed, I'll add that to the description.